pom.xml

```diff
@@ -113,6 +113,16 @@
       <artifactId>org.everit.json.schema</artifactId>
       <version>1.12.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>mod-pubsub-client</artifactId>
+      <version>1.1.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>util</artifactId>
+      <version>${rmb.version}</version>
+    </dependency>
```

Comment:
This is now being used in production code?

Reply:
@marcjohnson-kint Yes, we're using `HttpStatus` enum

Reply:
Hmm, ok, it seems a bit of a shame to include a whole new library dependency into the production artefacts for only a status enum